keras/callbacks/learning_rate_scheduler_test.py

```diff
@@ -12,7 +12,7 @@
 
 class LearningRateSchedulerTest(testing.TestCase):
     def setUp(self):
-        (x_train, y_train), _ = test_utils.get_test_data(
+        x_train, y_train, _, _ = test_utils.get_test_data(
```

Comment:
API wise, prefer keeping the original output grouping (2 tuples of 2 arrays) since it matches what you get from `keras.datasets`.